lib/model/rwfolder.go

```diff
@@ -200,9 +189,11 @@ func (f *sendReceiveFolder) Serve() {
 			if newHash := curIgnores.Hash(); newHash != prevIgnoreHash {
 				// The ignore patterns have changed. We need to re-evaluate if
 				// there are files we need now that were ignored before.
-				l.Debugln(f, "ignore patterns have changed, resetting prevVer")
+				l.Debugln(f, "ignore patterns have changed,",
```

Comment:
and I don't think you need to alter the comment as you are not adjusting anything it seems.